cms/grading/__init__.py

```diff
@@ -243,19 +243,26 @@ def get_compilation_commands(language, source_filenames, executable_filename,
         command += ["-XS", "-O2", "-o%s" % executable_filename]
         command += [source_filenames[0]]
         commands.append(command)
-    elif language == LANG_PYTHON:
+    elif language == LANG_PYTHON2:
         # The executable name is fixed, and there is no way to specify
         # the name of the pyc, so we need to bundle together two
         # commands (compilation and rename).
-        # In order to use Python 3 change them to:
-        # /usr/bin/python3 -m py_compile %s
-        # mv __pycache__/%s.*.pyc %s
         py_command = ["/usr/bin/python2", "-m", "py_compile",
                       source_filenames[0]]
-        mv_command = ["/bin/mv", "%s.pyc" % os.path.splitext(os.path.basename(
+        mv_command = ["/bin/mv", "%s.py2c" % os.path.splitext(os.path.basename(
```

Comment:
are you certain `py2c` is correct? Certainly locally, `python -m py_compile test.py` outputs `test.pyc` (which is what I expect), *not* `test.py2c`.